src/olympia/reviewers/templates/reviewers/addon_details_box.html

```diff
@@ -87,27 +86,26 @@ <h2>{{ addon_name }}</h2>
           {% endif %}
           {% if addon.has_listed_versions() %}
           <tr>
-            <th>{{ _('Visibility') }}</th>
+            <th>Visibility</th>
             <td>
               {% if not addon.is_disabled %}
-                {{ _('Visible') }}
+                Visible
               {% elif addon.is_disabled %}
-                {{ _('Invisible') }}
+                Invisible
               {% endif %}
             </td>
           </tr>
           {% endif %}
           {# XXX future feature: 'Localizations' ... #}
           <tr class="meta-rating">
-            <th>{{ _('Rating') }}</th>
+            <th>Rating</th>
             <td>
               <p class="addon-rating">
                 {% with num=addon.total_ratings %}
                   {% if num %}
                   {{ addon.average_rating|float|stars }}
                   {% with count='<span itemprop="count">{0}</span>'|format_html(num|numberfmt) %}
-                    {# Using num=count so we don't change an L10n string. #}
-                    {% set review_url_text = ngettext('{num} review', '{num} reviews', num)|format_html(num=count) %}
+                    {% set review_url_text = '{num} review(s)'|format_html(num=count) %}
```

Comment:
We decided we don't need "review"/"reviews" vs "review(s)"?

Reply:
I don't think it's worth it in reviewer tools